src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,146 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static int size;
+    private Node<T> firstNode;
+    private Node<T> lastNode;
+
+    public MyLinkedList() {
+        size = 0;
+    }
```

Comment:
you don't need it since size has its default value as 0

Reply:
if i delete size from constructor, i get errors 
don`t know how to avoid it

Reply:
lets remove this constructor